core/src/main/java/fr/sncf/osrd/cli/WorkerCommand.kt

```diff
@@ -44,10 +44,12 @@ class WorkerCommand : CliCommand {
     val WORKER_POOL: String
     val WORKER_REQUESTS_QUEUE: String
     val WORKER_ACTIVITY_EXCHANGE: String
+    val ALL_INFRA: Boolean
 
     init {
-        WORKER_ID = System.getenv("WORKER_ID")
-        WORKER_KEY = System.getenv("WORKER_KEY")
+        ALL_INFRA = !System.getenv("ALL_INFRA").isNullOrEmpty()
+        WORKER_ID = System.getenv("WORKER_ID") ?: (if (ALL_INFRA) "all_infra_worker" else null)
+        WORKER_KEY = System.getenv("WORKER_KEY") ?: (if (ALL_INFRA) "all" else null)
```

Comment:
This sets the priority as "env variable > all infra > null". I'd prefer if we used "all infra > env variable > null". It's quite minor but that way we could change a single flag to enable the "all infra" mode, instead of having to also unset the worker key/id.

```suggestion
        WORKER_ID =  if (ALL_INFRA) "all_infra_worker" else System.getenv("WORKER_ID")
        WORKER_KEY =  if (ALL_INFRA) "all" else System.getenv("WORKER_KEY")
```

Reply:
I went for the safety on regular use, but it's fine to change.
Any reaction to that is welcome to make a "definitive" choice: current choice is to go for @eckter's suggestion.

Reply:
Ah, I remember, I also wanted to let a possible choice of the `worker_key` name, which is not consistent with the "all infra" naming here.

The namings using "all" are about the targeted usage more than what it actually does/permit: no pre-load and work attached to just a grouping key, no matter the infra(s) being used.
Not sure it is useful for now or in the future to change/rename so I went for your suggestion in https://github.com/OpenRailAssociation/osrd/pull/9166/commits/302e4efbd7668b18a046b4ff58ea5f6bcb20e2ce after upvotes.